src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -24,10 +24,21 @@
     <!-- This script can't wait till the footer -->
     <script asp-name="admin-main" version="1" at="Head"></script>
 
-    <!-- TheAdmin & Bootstrap CSS -->
-    <style media="all" asp-name="admin" version="1"></style>
+    <style asp-name="TheAdminLayout" asp-src="~/TheAdmin/css/admin-layout.min.css" debug-src="~/TheAdmin/css/admin-layout.css" at="Head"></style>
+
+    @if (Orchard.IsRightToLeft())
+    {
+        <style asp-name="bootstrap-rtl" version="5" at="Head"></style>
```

Comment:
It could be simplified by ternary operator

Reply:
?

Reply:
Just leave it